### PR TITLE
Repo update

### PR DESCRIPTION
**Pull request by shanel262**
Added travis.yml and linting files. Updated package.json, license and readme

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+node_modules
+*.log
+.DS_Store
+
+dist
```
Comment (ckiss) [inline .eslintignore]: remove dist and client add new line as EOF

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
+![SenecaLogo][]
+> A [Senecajs org][] a seneca-auth plugin
```
Comment (ckiss) [inline README.md]: seneca-auth plugin?

File: lib/couchdb-store.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/*jslint node: true */
+/* jslint node: true */
```
Comment (ckiss) [inline lib/couchdb-store.js]: remove these jslint lines and copyright

File: lib/couchdb-store.js [hunk 4]
```diff
@@ -68,42 +63,41 @@ module.exports = function(opts) {
...
+  function configure (specification, cb) {
+    Assert(specification)
+    Assert(cb)
+
+    spec = specification
+
+    var conf = 'string' === typeof (spec) ? null : spec
+    if (!conf) {
+      conf = {}
+      var urlM = /^Couchdb:\/\/((.*?)@)?(.*?)(:?(\d+))$/.exec(spec)
```
Comment (ckiss) [inline lib/couchdb-store.js]: here you don't need to use capitalized name. it's an url format: `couchdb://`

File: lib/couchdb-store.js [hunk 5]
```diff
@@ -15,35 +15,31 @@
...
  */
 
-"use strict";
-var assert = require("assert");
-var _ = require('underscore');
-var uuid = require('uuid');
-var couchdb = require('felix-couchdb');
+'use strict'
+var Assert = require('assert')
+var _ = require('underscore')
```
Comment (ckiss) [inline lib/couchdb-store.js]: change this to require('lodash') everywhere you find underscore

**Comment by ckiss** [comment]
@shanel262 looks good, but I had some comments. after those are fixed I can review it again

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+language: node_js
+ +sudo: false
```
Comment (ckiss) [inline .travis.yml]: remove the +'es here and reformat the file please

File: .travis.yml [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+ +node_js:
+   - '5'
+   - '4'
+
+ cache:
+   directories:
+   - node_modules
+
+ after_script:
```
Comment (ckiss) [inline .travis.yml]: is _coverage_ a script in package.json? if not, you either add it there or remove it from here

File: README.md [hunk 8]
```diff
@@ -21,4 +30,37 @@ Usage:
...
+```sh
+npm install seneca-couchdb-store
+```
+
+## Test
+
+To run tests, simply use npm:
+
+```sh
+npm run test
```
Comment (ckiss) [inline README.md]: does this really work?

File: test/couchdb.test.js [hunk 9]
```diff
@@ -1,32 +1,31 @@
-/*jslint node: true */
+/* jslint node: true */
```
Comment (ckiss) [inline test/couchdb.test.js]: remove this jshint and copyright stuff

**Comment by ckiss** [comment]
@shanel262 just a few small comments. you're pretty much there

**Comment by ckiss** [comment]
LGTM @mihaidma can you have a look?

**Comment by mihaidma** [comment]
Looks ok. I'll merge. We'll have to check and fix tests if there are issues

**Comment by ckiss** [comment]
@mihaidma thanks. I'll get back to testing somewhere next week

**Pull request merged**